releasenotes/notes/add-back-lost-expvar-endpoint-7f801a71c1856cda.yaml

```diff
@@ -0,0 +1,14 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Restore missing `/debug/vars` expvar endpoint.
```

Comment:
We need to add the `APM: ` prefix for each of the reno entries relating to APM.

Reply:
Ok, added